ark/phenotyping/som_utils.py

```diff
@@ -1499,6 +1499,8 @@ def pixel_consensus_cluster(fovs, channels, base_dir, max_k=20, cap=3,
         index=False
     )
 
+    os.remove('Rplots.pdf')
```

Comment:
What about `cell_consensus_cluster`? Same output file?

Reply:
you're right cell clustering has the same issue